packages/cozy-jobs-cli/README.md

```diff
@@ -0,0 +1,24 @@
+`cozy-jobs-cli` provides binaries to launch konnectors or processes with
+environment variables that are used to connect to a cozy. Use this when
+you want a program to connect to a cozy.
+
+<!-- MarkdownTOC autolink=true -->
+
+- [`cozy-run-dev`](#cozy-run-dev)
+- [`cozy-konnector-dev`](#cozy-konnector-dev)
+
+<!-- /MarkdownTOC -->
+
+
+### `cozy-run-dev`
+
+- Provides `COZY_CREDENTIALS` to the underlying process
+
+You can use `COZY_URL` to connect to another cozy.
+
+### `cozy-konnector-dev`
+
+- Provides `COZY_CREDENTIALS` to the underlying process
+- Creates automatically `konnector-dev-config.json` and `importedData.json`
```

Comment:
`importedData` is only created by cozy-konnector-standalone (doctypes a filled in this file instead of the cozy).

I propose : 

- Provides `COZY_CREDENTIALS` to the locally running connector from `konnetor-dev-config.json` (which is created if it does not exist)